urwid/curses_display.py

```diff
@@ -515,8 +515,7 @@ def draw_screen(self, size, r ):
                 try:
                     if cs in ("0", "U"):
                         for i in range(len(seg)):
-                            self.s.addch( 0x400000 +
-                                ord(seg[i]) )
+                            self.s.addch( 0x400000 + seg[i] if PYTHON3 else ord(seg[i]) )
```

Comment:
`urwid.compat` seems to have `ord2` function defined for that purpose.

Reply:
Changed to ord2